com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -185,19 +180,14 @@ public class NetworkSceneManager : IDisposable
         /// When it is unset: After the scene has loaded, the PopulateScenePlacedObjects is called, and all NetworkObjects in the do
         /// not destroy temporary scene are moved into the active scene
         /// </summary>
-        internal static bool IsSpawnedObjectsPendingInDontDestroyOnLoad = false;
+        internal static bool IsSpawnedObjectsPendingInDontDestroyOnLoad;
 
         /// <summary>
         /// Client and Server:
-        /// Used for all scene event processing except for ClientSynchEventData specific events
+        /// Used for all scene event processing
         /// </summary>
-        internal SceneEventData SceneEventData;
+        internal Dictionary<Guid, SceneEventData> SceneEventDataQueue;
```

Comment:
"Queue" suffix implies `Queue<T>` by the name — what about `SceneEventDataStore` or something like that?